### DISCUSSION TITLE
Github Studente

### DISCUSSION THREAD

**Discussion by alexnogueirasilva** [community/community#131963] [GitHub Education]
### Hi everyone,

Benefits for being a student have not yet been released

**Comment by Aditijainnn** [comment]
> ### Hi everyone,
> 
> Benefits for being a student have not yet been released

Contact GitHub Support

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
